### ISSUE TITLE
Things are invisible

### ISSUE THREAD

**Issue by w1redl4in** [szorfein/dots#4]
Just followed your steps, everything looks fine but when i log in awesomeWM every window is invisible, can only see silhouette

**Comment by szorfein** [comment]
I need more infos to reproduce this, distro, screenshot. It sounds like a problem with the compositor or video driver.

**Comment by w1redl4in** [comment]
Distro: Ubuntu 20.04 LTS

Return of `chezmoi apply` 
![image](https://user-images.githubusercontent.com/43390533/95081748-6321a880-06f0-11eb-9fa9-112c169809a1.png)

When i log in on awesomeWM:
![image](https://user-images.githubusercontent.com/43390533/95081792-76cd0f00-06f0-11eb-84ea-2e70306545cc.png)

**Comment by szorfein** [comment]
Thanks, with Ubuntu, it should be the version of awesome, you need the version 4.3:

    $ awesome --version
    awesome v4.3

Unfortunately, i have no idea how install this on Ubuntu, maybe look unstable package...

**Comment by w1redl4in** [comment]
```
awesome --version
awesome v4.3 (Too long)
 • Compiled against Lua 5.3.3 (running with Lua 5.3)
 • D-Bus support: ✔
 • execinfo support: ✔
 • xcb-randr version: 1.6
 • LGI version: 0.9.2
```


Oh, thats sad haha

**Comment by szorfein** [comment]
Ok for the version. I have found a post [here](https://stackoverflow.com/questions/57084569/how-do-i-install-awesome-wm-on-ubuntu-19-04). 

You can try to change the file `~/.dmrc` with this:

    Session=awesome --no-argb

And logout and back

**Comment by w1redl4in** [comment]
Still invisible.

Theres a hotkey that i can logout? I always get stuck on awesomeWM

**Comment by szorfein** [comment]
Mod4 + Echap + e :) i look on https://github.com/awesomeWM/awesome if they have similar issue.  
EDIT: You can found a log file too at ~/.awesome-errors , it can be usefull.

**Comment by szorfein** [comment]
No idea here :+1: , can you post all packages installed on your system (with `apt list --installed` or `dpkg -l`) 
and your log file `~/.xsession-errors` (if exist)

**Comment by szorfein** [comment]
I think, maybe it lack some dependencies for Xorg like `xorg-xrdb` , colors are just not loaded from the Xresources.
EDIT: It can come from your Diplay Manager too which prevents the .Xresources from loading.

**Comment by szorfein** [comment]
should be solved with 333e9fb
